docs/release-notes/8.0.42-33.md

```diff
@@ -0,0 +1,115 @@
+
+* [`--rocksdb_compaction_readahead_size`](../variables.md#rocksdb_compaction_readahead_size) - the default value is changed from `0` (zero) to `2097152`.
+
+**Deprecates MyRocks variable**
+
+* [`--rocksdb_disable_instant_ddl`](../variables.md#rocksdb_disable_instant_ddl) - this variable is being deprecated and is expected to be removed in a future release.
+
+**Removes MyRocks variables**
+
+* [`--rocksdb-access-hint-on-compaction-start`](../variables.md#rocksdb_access_hint_on_compaction_start)
+* [`--rocksdb_large_prefix`](../variables.md#rocksdb_large_prefix)
+* [`--rocksdb_strict_collation_check`](../variables.md#rocksdb_strict_collation_check)
+* [`--rocksdb_strict_collation_exceptions`](../variables.md#rocksdb_strict_collation_exceptions)
+
+### MySQL 8.0.42
+
+Improvements and bug fixes provided by Oracle for MySQL 8.0.42 and included in Percona Server for MySQL are the following:
+
+* Fixed an issue where `CHECK TABLE` sometimes incorrectly reported that spatial indexes were corrupted. (Bug #37286473)
+
+* Fixed an issue in InnoDB redo log recovery to improve data safety after a crash. (Bug #37061960)
+
+* Fixed an issue where reading `index_id` values could lead to incorrect behavior with indexes. (Bug #36993445, Bug #37709706)
+
+* Fixed a bug related to the `lower_case_table_names` setting that caused inconsistent behavior with table names on different systems. (Bug #32288105)
+
+Find the complete list of bug fixes and changes in the [MySQL 8.0.42 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-42.html).
+
+## Improvements
+
+* [PS-9024](https://perconadev.atlassian.net/browse/PS-9024): Improves the `audit_log_filter_set_user()` UDF to accept account names with wildcard characters (`'%'` and `'_'`) in the host part.
+
+* [PS-9561](https://perconadev.atlassian.net/browse/PS-9561): Updates the C++ level of the KMPI library to enhance error handling capabilities.
+
+* [PS-9765](https://perconadev.atlassian.net/browse/PS-9765): Updates the list of MyRocks variables. You can find the list of variables in [MyRocks server variables](../variables.md).
+
+## Bug Fixes
+
+* [PS-9033](https://perconadev.atlassian.net/browse/PS-9033): The `audit_log_filter` plugin did not register remote accesses.
+
+* [PS-9390](https://perconadev.atlassian.net/browse/PS-9390): In some cases, using `JSON_TABLE` inside an `IN` or `EXISTS` subquery caused incorrect results. This happened when the subquery referred to a table from the main query, and a semijoin optimization was applied. Percona merged the fix from [MySQL](https://github.com/mysql/mysql-server/commit/15e6812b80 ).
+
+* [PS-9609](https://perconadev.atlassian.net/browse/PS-9609): The `audit_log_filter` plugin could not be installed when Percona Server was using `component_keyring_kmip`.
+
+* [PS-9628](https://perconadev.atlassian.net/browse/PS-9628): The `binlog_encryption` did not work with `component_keyring_kmip`.
+
```

Comment:
One more KMIP-related issue is missing
PS-9661
Although marked as "INTERNAL", still makes sense to mention it here.

Reply:
added